pkg/api/object.go

```diff
@@ -20,11 +20,11 @@ package api
 import (
 	"fmt"
 	"io/ioutil"
+	"net/http"
 	"sort"
 
 	"github.com/megaease/easegress/pkg/supervisor"
 
-	"github.com/kataras/iris"
```

Comment:
Inline #63, #68, #78..., the ` ObjectPrefix + "/{name:string}"` this iris path grammar won't work in go-chi. Plz read the go-chi's example in [README](https://github.com/go-chi/chi/blob/master/README.md) carefully!

It should be `ObjectPrefix +"/{name}". The other RESTful APIs in this directory or in `mesh controller`  also need to be fixed.